solutions/USACO feb21-clockwise-fence.mdx

```diff
@@ -9,7 +9,12 @@ author: Melody Yu
 
 ## Solution (Vector Cross Product)
 
-The vector cross product can be used to determine if 2 vectors are rotating clockwise or anticlockwise. For this question, we treat each step that Farmer John walks as a vector and take the cross product of the current step and the previous step. If we sum all the cross product values, we can determine if the overall direction is CW or CCW based on the sign of the sum.
+To find the area of a triangle with a vertex at (0, 0), a second vertex at (x1, y1) and third vertex at (x2, y2), we can use the formula (x1y2 - y1x2)/2. This value is positive if x1/y1 is greater than x2/y2, since the slope becomes smaller, making it clockwise. Likewise, a negative area means the direction is counterclockwise.
+
+In the program, every time Farmer John lays down another fence, a new triangle is formed with his new position, the origin, and the previous position (the last fence he laid down). We can calculate the area of this new triangle. After he lays down all the fences, we can sum the areas of the multiple triangles to find if the final direction was positive (clockwise) or negative (counterclockwise).
+
+In terms of physics, [cross product](https://en.wikipedia.org/wiki/Cross_product) is twice the area of the triangle between the two vectors (positive/negative). Every time Farmer John lays down a fence, we can treat it as a vector and compute the cross product of this vector against the previous one. By adding the cross products together, we get twice the final area of the polygon formed by these vectors. Based on the positive/negative value of the area, we can know if it's counterclockwise or clockwise (vector directing up or down with the right hand rule). The process is also how the shoelace formula is proved.
```

Comment:
not just physics... cross product is also math

Reply:
Maybe just remove the "In terms of physics" entirely.

Reply:
Yep, removing "In terms of physics" is fine

Reply:
got it, thanks